website/migratefile

```diff
@@ -0,0 +1,237 @@
+advanced/tenancy.md -> sys-mgmt/tenancy.md
```

Comment:
Should this be committed?

Reply:
Yes, I think it is OK because it is only moving the file to a new location, but, just like before, it is not in the Table of Contents (`sidebar.js`) so no one knows it is there. This is basically the same as it is in current docs, just moving the location of the Tenancy docs.